entrypoint_prod.sh

```diff
@@ -1 +1 @@
-poetry run gunicorn -w 4 -b 0.0.0.0:$PORT todo_app.app:app --preload
\ No newline at end of file
+poetry run gunicorn -w 4 -b 0.0.0.0:$PORT 'todo_app.app:create_app()' --preload
```

Comment:
Minor, but worth making sure you always have a shebang at the top of your script files (e.g. `#!/bin/bash`) so that it's clear what shell should execute the file

Reply:
I have added shebang as advised